blenderproc/command_line.py

```diff
@@ -60,6 +60,7 @@ def cli():
     format_dict = lambda d : '\n'.join("{}: {}".format(key, value) for key, value in d.items())
     parser_pip.add_argument('pip_mode', choices=options['pip'], help=format_dict(options['pip']))
     parser_pip.add_argument('pip_packages', metavar='pip_packages', nargs='*', help='A list of pip packages that should be installed/uninstalled. Packages versions can be determined via the `==` notation.')
+    parser_pip.add_argument('--not-use-custom-package-path', dest='not_use_custom_package_path', action='store_true', help='If set, the pip packages will not be installed into the separate custom package folder, but into blenders python site-packages folder.')
```

Comment:
Could you add some details here why this might be useful sometimes?

Reply:
Done :+1: